bin/release

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+GEM_VERSION=$(ruby -e '$LOAD_PATH << File.join(File.dirname(__FILE__), "lib"); require "nxt_http_client/version"; print NxtHttpClient::VERSION')
+
+gem build nxt_http_client.gemspec
+
+if [ ! -d "pkg" ]; then
+  mkdir pkg
+fi
+
+mv nxt_http_client-$GEM_VERSION.gem pkg/nxt_http_client-$GEM_VERSION.gem
+
+git tag -m "Release $GEM_VERSION" -a v$GEM_VERSION HEAD
+git push origin --tags
+
+gem push --key github --host https://rubygems.pkg.github.com/nxt-insurance pkg/nxt_http_client-$GEM_VERSION.gem
```

Comment:
@Thrashattack here your'e pushing to the github package registry, not to rubygems.org. That's what we want for nxt_clients, but for nxt_http_client you need to change the host.

Reply:
The `rake release` command will push it to RubyGems. 

I'll change the Readme to explicitly mention both hosts and procedures to push to GPR and RubyGems.

Reply:
> I'll change the Readme to explicitly mention both hosts and procedures to push to GPR and RubyGems.

@Thrashattack did you do it already? Readme is still focused on github rather than rubygems.

Reply:
Yes, I added the GitHub section, and let it as-is the existent guidelines about RubyGems

Reply:
But why we need Github section here, if we release this gem to Rubygems? :)

Reply:
FYI https://getsafe.slack.com/archives/C03DT3HD953/p1708607961630189 

Long story short: We might wanna have a single source of truth when it comes to Gems hosting. 
And RubyGems process seems a bit difficult. Also releasing it to GPR and at the same time to RubyGems doesn't seem to be a problem. 

Further concerns are to be discussed in BE weekly about moving all our gems to GPR or not.